docker/nginx/conf.d/include/location-skylink

```diff
@@ -0,0 +1,98 @@
+include /etc/nginx/conf.d/include/cors;
+include /etc/nginx/conf.d/include/proxy-buffer;
+include /etc/nginx/conf.d/include/proxy-cache-downloads;
+include /etc/nginx/conf.d/include/track-download;
+
+# redirect purge calls to separate location
+error_page 462 = @purge;
+if ($request_method = PURGE) {
+    return 462;
+}
+
+limit_conn downloads_by_ip 100; # ddos protection: max 100 downloads at a time
```

Comment:
this is a limit applied cross all account types?

Reply:
yes, it's not a new one though, been there for a while